### PR TITLE
Got buttons to render. and most of functionality of buttons

### PR DESCRIPTION
**Pull request by charleyarby**

### CODE REVIEW

File: client/src/components/PageButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    super(props);
+    this.state ={
+      pageButtons: []
+    }
+    this.setButtons = this.setButtons.bind(this);
+
+  }
+
+
+  setButtons() {
```
Comment (sreeshman1) [inline client/src/components/PageButtons.jsx]: what is set buttons being used for here?

Comment (charleyarby) [reply client/src/components/PageButtons.jsx]: left over code. thanks. it doesn't do anything. I'm going to take it off

File: client/src/components/Rating.jsx [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+      </div>
+    </div>
+  </div>
+
+  )
+    }
+
+export default Rating;
+
+
```
Comment (sreeshman1) [inline client/src/components/Rating.jsx]: i think it would be more organized to have the css here in a different file

File: server/index.js [hunk 3]
```diff
@@ -3,10 +3,25 @@ let port = 3001;
 let app = express();
 const db = require('../database/index.js')
 
-app.use(express.static(__dirname+ '/../client/dist'));
 
-app.get('/0', (req, res) => {
-  db.getAll((err, data)=>{
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
+  next();
```
Comment (sreeshman1) [inline server/index.js:9]: theres a node module called cors that you can use to do this part; i think it would be cleaner

**Review by sreeshman1 — APPROVED** [review]
I really like the way you format your code and name your variables and functions; it was really easy to read your code and understand what each line did.

**Pull request merged**